.github/workflows/test.yml

```diff
@@ -22,4 +22,6 @@ jobs:
           key: v1/${{ runner.os }}/node-${{ matrix.node }}/${{ hashFiles('**/yarn.lock') }}
           restore-keys: v1/${{ runner.os }}/node-${{ matrix.node }}/
       - run: yarn
+      - run: git clone https://github.com/percy/cli;cd cli;yarn global:link;cd ..;yarn remove @percy/cli && yarn link @percy/cli
```

Comment:
* make the `run:` multi line for readibility 
* clone in other `/tmp` maybe 
* `yarn build` is missed
* `yarn link @percy/cli` does this link all the mono repo packages?
* can you test this with https://github.com/nektos/act?